autogluon/task/tabular_prediction/predictor.py

```diff
@@ -341,6 +341,64 @@ def fit_summary(self, verbosity=3):
             print("*** End of fit() summary ***")
         return results
 
+    def transform_features(self, dataset=None, model=None, base_models=None):
+        """
+        Transforms dataset features through the AutoGluon feature generator.
+        This is useful to gain an understanding of how AutoGluon interprets the dataset features.
```

Comment:
Should clarify that: Individual AutoGluon models like the neural network may apply additional feature transformations that are not reflected in this method, which only applies universal transforms employed by all AutoGluon models.

Reply:
Added